plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -124,12 +124,25 @@ public static function embed_page_header() {
 
 		$sections = self::get_embed_breadcrumbs();
 		$sections = is_array( $sections ) ? $sections : array( $sections );
+
+		$page_title      = '';
+		$pages_with_tabs = array( 'Settings', 'Reports', 'Status' );
```

Comment:
Will this cause issues in the future when a page with tabs is added or tabs to existing page? Maybe not, but wanted to raise the issue

Reply:
Good question! I think it will be fine. We should add them to this list to show the correct page title when new pages have tabs or tabs to existing pages. But if someone forgets to add the tab to the list, it will still show the current page title.

Reply:
@chihsuan Do I read it right that we compare localized breadcrumbs to hardcoded English titles here?

Reply:
@tomalec I think you're right! Good catch. 👍 Would you like to take a stab at fixing it? Otherwise, I'll add it to Ghidorah backlog.

Reply:
Thanks :) I made a fix at https://github.com/woocommerce/woocommerce/pull/51606; I'd appreciate a review, as I'm not 100% familiar with that code and may miss something important. Plus, I wonder why the menu items are not translated into any language I have checked :thinking:

Reply:
Thanks for fixing it so quickly! @tomalec The change looks good to me. 

I can see the menu items translated into the languages I checked. I'm not sure why you don't see them. Maybe you need to update the translations? Go to **WP Admin > Dashboard > Updates and click the Update Translations** button. Sometime I forget to do that and it causes issues like this.

Reply:
> Maybe you need to update the translations? 

Thanks that was it :)